src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/MethodWithGCInfo.cs

```diff
@@ -13,7 +13,7 @@ namespace ILCompiler.DependencyAnalysis.ReadyToRun
 {
     public class MethodWithGCInfo : ObjectNode, IMethodCodeNode, IMethodBodyNode
     {
-        public readonly MethodGCInfoNode GCInfoNode;
+        public MethodGCInfoNode GCInfoNode;
```

Comment:
Maybe I'm mis-reading but it looks like we can still use `readonly` here.

Reply:
For sure, restored.